docs/api/synchronization.rst

```diff
@@ -0,0 +1,160 @@
+.. _api-synchronization:
+
+Synchronization
+###############
+
+This section describes the basic aspects of synchronization using *Kinto*.
+
+.. note::
+
+    If you are looking for a ready-to-use synchronization solution,
+    jump to :ref:`sync-implementations`.
+
+
+The main idea consists in maintaining a local database up-to-date
+with the *Kinto* server. Remote changes are downloaded and applied
+on the local data. Local changes are uploaded using HTTP headers to control
+concurrency and overwrite.
+
+In short:
+
+#. Poll for remote changes using ``?_since=<timestamp>``
+#. Apply changes locally
+#. Post creations
+#. Use concurrency control to send updates and deletes
+
+
+Polling for changes
+===================
+
+When fetching the collection records with ``?_since=<timestamp>``, *Kinto* returns
+every records that were created/updated/deleted **after** this timestamp.
+
+#. Cold sync → no ``?_since``.
```

Comment:
This list is succinct to point of being hard to understand. I'm not sure how to suggest an amelioration since I don't entirely grok what's being said. :confused: Maybe try being a bit more verbose with this and then we'll review it together.